cf-editor-rte-toolbar-button-sample/src/aem-cf-editor-1/web-src/src/utils.js

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
Do we need this file?

Reply:
Please look at [this comment](https://github.com/adobe/aem-uix-examples/pull/7#issuecomment-1863455698)